fennel/internal_lib/to_proto/to_proto.py

```diff
@@ -473,12 +478,12 @@ def _extractor_to_proto(
                 inputs.append(feature_to_proto_as_input(f))
         elif isinstance(input, Featureset):
             raise TypeError(
-                f"Extractor input {input} is a Featureset, please use a"
+                f"{extractor_name} Extractor input {input} is a Featureset, please use a"
```

Comment:
nit: now that 'Extractor' isn't the first letter of the sentence, make it start with small case. Also, use small case for 'Featureset' & DataFrame later.

Reply:
changed